how-to-enable-trace-listener-for-asp-net-core-projects.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Problems in ASP.NET Core applications
```

Comment:
How about a title like "Troubleshooting reporting implementation into ASP.NET Core application". I think it will be easier to discover from google.